src/pricing-fetcher.ts

```
@@ -32,13 +45,23 @@ export class PricingFetcher implements Disposable {
 		}
 
 		try {
-			logger.warn('Fetching latest model pricing from LiteLLM...');
-			const response = await fetch(LITELLM_PRICING_URL);
-			if (!response.ok) {
-				throw new Error(`Failed to fetch pricing data: ${response.statusText}`);
+			let data: unknown;
+			const isUrl = this.pricingSource.startsWith('https://');
+
+			if (isUrl) {
+				logger.warn(`Fetching model pricing from URL: ${this.pricingSource}`);
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)